src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -1458,27 +1453,8 @@ namespace SurfaceGeometry {
                 // Find and replace negative SubSurfNum with new MovedSurfs num in surface list for reporting
                 std::replace(DataSurfaces::AllSurfaceListReportOrder.begin(), DataSurfaces::AllSurfaceListReportOrder.end(), -SubSurfNum, MovedSurfs);
             }
-            for (int SubSurfNum = 1; SubSurfNum <= TotSurfaces; ++SubSurfNum) {
 
-                if (SurfaceTmpClassMoved(SubSurfNum)) continue;
-                if (state.dataSurfaceGeometry->SurfaceTmp(SubSurfNum).Zone != ZoneNum) continue;
-                if (state.dataSurfaceGeometry->SurfaceTmp(SubSurfNum).ExtBoundCond <= 0) continue; // Interior
-                if (state.dataSurfaceGeometry->SurfaceTmp(SubSurfNum).Class == SurfaceClass::Window) continue;
-                if (state.dataSurfaceGeometry->SurfaceTmp(SubSurfNum).Class == SurfaceClass::GlassDoor) continue;
-                if (state.dataSurfaceGeometry->SurfaceTmp(SubSurfNum).Class == SurfaceClass::TDD_Diffuser) continue;
-                if (state.dataSurfaceGeometry->SurfaceTmp(SubSurfNum).Class == SurfaceClass::TDD_Dome) continue;
-
-                ++MovedSurfs;
-                Surface(MovedSurfs) = state.dataSurfaceGeometry->SurfaceTmp(SubSurfNum);
-                SurfaceTmpClassMoved(SubSurfNum) = true; // 'Moved'
-                // Reset BaseSurf to it's positive value (set to negative earlier)
-                Surface(MovedSurfs).BaseSurf = -Surface(MovedSurfs).BaseSurf;
-                state.dataSurfaceGeometry->SurfaceTmp(SubSurfNum).BaseSurf = -1;
-                // Find and replace negative SubSurfNum with new MovedSurfs num in surface list for reporting
-                std::replace(DataSurfaces::AllSurfaceListReportOrder.begin(), DataSurfaces::AllSurfaceListReportOrder.end(), -SubSurfNum, MovedSurfs);
-            }
-
-            // The window subsurfaces (includes SurfaceClass::TDD_Diffuser)
+            // The exterior window subsurfaces (includes SurfaceClass::Window and SurfaceClass::GlassDoor) goes next
```

Comment:
Remember to edit the comment block at the top of this function to be current and it should mention the various first/last variables.